test/TesterInternal/General/RequestContextTest.cs

```diff
@@ -169,22 +181,33 @@ public void Dispose()
 
             IRequestContextTestGrain grain = this.fixture.GrainFactory.GetGrain<IRequestContextTestGrain>(GetRandomGrainId());
 
+#if NETSTANDARD
+            RequestContext.ActivityId.Value = activityId;
+#else
             Trace.CorrelationManager.ActivityId = activityId;
+#endif
            Assert.Null(RequestContext.Get(RequestContext.E2_E_TRACING_ACTIVITY_ID_HEADER));
             Guid result = await grain.E2EActivityId();
             Assert.Equal(activityId,  result);  // "E2E ActivityId not propagated correctly"
             RequestContext.Clear();
 
+#if NETSTANDARD
+            RequestContext.ActivityId.Value = nullActivityId;
```

Comment:
agreed, we still need to think about back-compat implications of this. We don't want to blindly switch to AsyncLocal. We did on .NET Core because we didn't have an alternative, but not sure what it would mean for the 1.5 release that still only targets .NET.